contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -27,7 +28,7 @@
 from pex.pex_info import PexInfo
 
 
-class MypyTask(ResolveRequirementsTaskBase):
+class MypyTask(LintTaskMixin, ResolveRequirementsTaskBase):
   """Invoke the mypy static type analyzer for Python."""
 
```

Comment:
Should probably add a `deprecated_options_scope = 'mypy'` line, to handle the fact that this moved.